### PR TITLE
feat: add v8

### PR DESCRIPTION
**Pull request by gocarlos**
copied from https://github.com/extcpp/conan-v8

this is a WIP as v8 seems pretty complex to package and its not clear how to do it with the current conan workflow:

as per https://github.com/microsoft/vcpkg/issues/372#issuecomment-340123529 we would need to download two artifacts, in conan you have to specify one in the conandata.yml file

- [ ] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [ ] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [ ] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [ ] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

closes https://github.com/conan-io/conan-center-index/issues/995

cc/ @ObiWahn

### CODE REVIEW

File: recipes/v8/all/conandata.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+sources:
+  "8.2.49":
+    sha256: bf61bc7ffe556b30464ac0734e18d975fe6a8676d9c2fcb7c215981a46381847
+    url: https://github.com/bacnet-stack/bacnet-stack/archive/4a916468c63de478b84ef4d0c67d541cd84da27e.zip
```
Comment (gocarlos) [inline recipes/v8/all/conandata.yml:4]: TODO

**Comment by Croydon** [comment]
I'm not sure if it works at all or how good the quality is, but here is another v8 recipe: https://github.com/inexorgame/conan-v8

**Comment by prince-chrismc** [comment]
> we would need to download two artifacts, in conan you have to specify one in the conandata.yml file

I came across doxygen recipe which actually downloads four artifacts! Something similar might be possible here

**Comment by gocarlos** [comment]
> I'm not sure if it works at all or how good the quality is, but here is another v8 recipe: https://github.com/inexorgame/conan-v8

https://github.com/extcpp/conan-v8 is a fork of this one

**Comment by ObiWahn** [comment]
I think it was working ok, but I ran out of build time on travis. Therefore I gave up for the time. If somebody is willing to provide me with CI time I would look into it again.

**Comment by gocarlos** [comment]
> I think it was working ok, but I ran out of build time on travis. Therefore I gave up for the time. If somebody is willing to provide me with CI time I would look into it again.

you could try to contribute to CCI as we do not have this issue here :)
furthermore, in the long term, this is the place to put it :D

**Comment by Croydon** [comment]
Also GitHub Actions can run up to 6 hours https://docs.github.com/en/actions/getting-started-with-github-actions/about-github-actions#usage-limits

**Pull request closed without merging**